base.js

```diff
@@ -289,5 +289,17 @@ module.exports = {
 
         // LWC import validation
         '@lwc/lwc/no-disallowed-lwc-imports': 'error',
+
+        // Disable any direct importing of LDS artifacts generated by the LWC compiler
+        'no-restricted-imports': [
```

Comment:
Should we be more explicit with this rule name? I guess I could see this being a good place to group multiple patterns for restricted imports

Reply:
This is just a built-in rule for eslint, so I didn't want to reinvent the wheel. We can use a custom error message if we really want to though

Reply:
A custom error message sounds better